### DISCUSSION TITLE
waline服务端支持部署在21云盒子吗？

### DISCUSSION THREAD

**Discussion by AkarinLiu** [walinejs/waline#477] [Q&A] [answered]
@lizheming 我想放弃Vercel了，因为稳定性水土不服，所以我想换到21云盒子，希望有大佬能支持服务端部署在21云盒子。

**Comment by Mister-Hope** [comment, marked as answer]
从官网上看这是一个商业化的服务提供商，我并没有看到免费额度。

https://www.21cloudbox.com/pricing.html

如果它有，希望你能具体描述一下并提供相关的官方页面，我们会评估。

另外如果它和 Vercel 提供相似的服务，你应该能够按照 Vercel 的部署方式直接部署。如果它的确和 Vercel 相似，而且你在部署方面遇到了问题，欢迎提供具体信息。

**Reply by AkarinLiu** [reply]
我问了它们的客服，直到现在一直没有回复我
